winterfell/src/tests.rs

```diff
@@ -0,0 +1,206 @@
+use super::*;
+use prover::{
+    crypto::{hashers::Blake3_256, DefaultRandomCoin},
+    math::{fields::f64::BaseElement, FieldElement},
+    matrix::ColMatrix,
+};
+
+#[cfg(not(feature = "std"))]
+use alloc::{vec, vec::Vec};
```

Comment:
Note: I wasn't able to make the imports work when using `winter-utils` as a `dev-dependency`, so I opted for this